pages/mesosphere/dcos/services/data-science-engine/1.0.1/quick-start/index.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: layout.pug
+navigationTitle: Quick Start Guide
+excerpt: Get up and running with DC/OS Data Science Engine
+title: Quick Start
+menuWeight: 2
+model: /mesosphere/dcos/services/data-science-engine/data.yml
+render: mustache
+enterprise: true
+
+---
+
+This guide explains how to install the {{ model.techName }} Service.
+
+# Prerequisites
+
+
+- DC/OS and DC/OS CLI installed with a minimum of {{ model.nodeDescription }}.
+- Depending on your security mode, {{ model.techName }} requires service authentication for access to DC/OS. See [Provisioning a service account](/mesosphere/dcos/services/data-science-engine/1.0.1/security/#provisioning-a-service-account) for more information.
+
+<p class="message--note"><strong>NOTE: </strong> If you are planning to use <a href="https://docs.d2iq.com/mesosphere/dcos/services/data-science-engine/1.0.1/integrations/hdfs/">HDFS</a> on {{ model.techName }}, you will need a minimum of five nodes.</p>
+
+| Security Mode | Service Account |
+|----------------|------------------|
+| Disabled | Not available |
+| Permissive | Optional |
+| Strict | Required |
+
+The following points should be considered before installing the service:
```

Comment:
Done. I have moved the warning note before the HDFS note, and have styled it as a note. If you want this to be stronger, such as "caution" or even "warning", let me know.

Reply:
Thanks, Sarah. I think we should make it at least a caution (or warning) because we had a severe incident with a customer which was caused by this issue. I don't know what the criteria are for each of these types of messages but I'm sure it should be stronger than just a note.